cli/tests/integrations/test_config.py

```diff
@@ -12,20 +12,24 @@
 
 @pytest.fixture
 def env():
-    return {
+    r = os.environ.copy()
```

Comment:
Hmm.   This might be OK, but can you first figure out which missing env var is causing this failure?